VLogProcessor/src/edu/umich/robustopt/vertica/VerticaLatencyMeter.java

```diff
@@ -96,7 +96,12 @@ public void emptyCache() throws Exception {
 		while (nAttempts <= 2) {
 			int status = -1;
 			rc = 0;
-			rs = stmt.executeQuery(sql);
+			try {
+				rs = stmt.executeQuery(sql);
+				log.status(LogLevel.STATUS, "Using academic version, barzan_empty_cache() has been called");
```

Comment:
Here I catch the exception when `barzan_empty_cache()` does not exist in vertica